bip-0008/states.dot

```diff
@@ -7,7 +7,7 @@ digraph {
   "DEFINED" -> "STARTED" [label="height >= start_height"];
   "STARTED" -> "MUST_SIGNAL" [label="height + 2016 >= timeoutheight AND lockinontimeout"];
   "STARTED" -> "FAILED" [label="height >= timeoutheight\nAND\nNOT lockinontimeout"];
-  "LOCKED_IN" -> "ACTIVE" [label="always"];
+  "LOCKED_IN" -> "ACTIVE" [label="height >= minimum_activation_height"];
```

Comment:
Should be a `"LOCKED_IN":se -> "LOCKED_IN":ne;` loop edge since it can stay in LOCKED_IN for multiple periods now.

Reply:
I've added the loop edge